.travis.yml

```diff
@@ -6,6 +6,7 @@ rvm:
   - 2.0.0
   - 2.1
   - 2.2
+  - 2.3.0
```

Comment:
@bbatsov , doesn't seem to work yet with 2.3 as travis doesn't provide binary for it yet...

Shall we wait that travis provide binary or use 2.3.0 meanwhile ?

```
 rvm use 2.3 --install --binary --fuzzy

ruby-2.3 is not installed - installing.

Searching for binary rubies, this might take some time.

Requested binary installation but no rubies are available to download, consider skipping --binary flag.

Gemset '' does not exist, 'rvm ruby-2.3 do rvm gemset create ' first, or append '--create'.

The command "rvm use 2.3 --install --binary --fuzzy" failed and exited with 2 during .
```